src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let result = {};
+  const check = [...new Set(Object.values(robot))];
+
+  if (check.length === 0) {
+    result = {};
+  } else if (check.length !== Object.values(robot).length) {
+    return null;
+  } else {
```

Comment:
[CODE STYLE] - You should return the value immediately in if statements. And don't use else after return statement. If return is performed - code after it won't be executed anyway.